lua/wikis/apexlegends/Infobox/Team/Custom.lua

```diff
@@ -42,13 +42,9 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
----@return string?
+---@return Widget
 function CustomTeam:createBottomContent()
-	return Template.expandTemplate(
-		mw.getCurrentFrame(),
-		'Upcoming and ongoing tournaments of',
-		{team = self.name or self.pagename}
-	)
+	return UpcomingTournaments{name = self.name or self.pagename}
```

Comment:
`self.name or self.pagename` doesn't make sense here (Infobox Basic already does this).
should rather be `self.teamtemplate` plus fetching the page for that

Applies to all team customs, not just apex - they should all use the same thing to pass to UpcomingTournaments